Documents/local-dependencies.md

```diff
@@ -0,0 +1,99 @@
+# Dev Dependencies
+
+This project depends on several services, which all need to be running in order for the project to operate fully.
+
+Assuming you have Docker-Compose installed, the easiest way to get all of the platform's dependencies up and running with the provided docker-compose dev setup:
+
+```shell
+docker-compose -f docker-dev.yml up
+```
+
+Alternatively, you can start each one individually with the following instructions.
+
+
+## Write Journal
+
+The write side of this system is supported by either a local `leveldb` database or Cassandra.
+
+#### leveldb
+
+`leveldb` is best for local environments and for smoke testing. When using `leveldb`, the project automatically loads sample data into the environment when it starts up.
+
+To use it, set the following environment variable in the same session where you run the project in sbt:
+
+```shell
+export HDMA_IS_DEMO=true
+```
+
+
+#### Cassandra
+
+If using Cassandra is desired, set the following environment variable:
+
+```shell
+export HDMA_IS_DEMO=false
+```
+
+The easiest way to run a Cassandra server to support this application for testing is to do it through Docker:
+
+```shell
+docker run --name cassandra -p 9042:9042 -p 7000:7000 -p 7199:7199 cassandra:3.10
+```
+
+If you want to connect to this server, the following `docker` command will give you access to the Cassandra instance started in the previous step:
+
+```shell
+docker run -it --link cassandra:cassandra --rm cassandra cqlsh cassandra
+```
+
+Once the `Cassandra` server is running, set the following environment variable to the appropriate Cassandra host (in this example, the default local docker host for a machine running MacOs X):
+
+```shell
+export CASSANDRA_CLUSTER_HOSTS=192.168.99.100
+```
+
+To load data into `Cassandra`, you can run the following (the Cassandra server needs to be running and correct environment variables configured as per the previous instructions):
```

Comment:
I would just link the panel docs for the instructions here.  If a user is following this doc, they'll miss the environment variable they need to set for the panel loader to run properly.